Cargo.toml

```diff
@@ -32,6 +32,7 @@ wsts = { version = "13.0.1" }
 
 # Crates.io
 aquamarine = { version = "0.6.0", default-features = false }
+assert_matches = "1.5.0"
```

Comment:
Where is this used, seems like it can be removed.

Reply:
Is/was used in tests, need to see which tests are still portable to this PR's refactors and if they'll make use of it or not.

Note: stdlib recommends `assert_matches!()` in general over `assert!(matches!())` because the latter loses context upon failure, i.e. you only see that the `matches!()` returned false and not why.